interface/js_binding/ktx_wrapper.cpp

```diff
@@ -457,36 +419,22 @@ EMSCRIPTEN_BINDINGS(ktx)
         .value("PVRTC2_4_RGBA", KTX_TTF_PVRTC2_4_RGBA)
         .value("ETC", KTX_TTF_ETC)
         .value("EAC_R11", KTX_TTF_ETC2_EAC_R11)
-        .value("EAC_RG11", KTX_TTF_ETC2_EAC_RG11)
-    ;
+        .value("EAC_RG11", KTX_TTF_ETC2_EAC_RG11);
     enum_<ktx_transcode_flag_bits_e>("TranscodeFlagBits")
-        .value("TRANSCODE_ALPHA_DATA_TO_OPAQUE_FORMATS",
-               KTX_TF_TRANSCODE_ALPHA_DATA_TO_OPAQUE_FORMATS)
-    ;
-    enum_<ktxOrientationX>("OrientationX")
-        .value("LEFT", KTX_ORIENT_X_LEFT)
-        .value("RIGHT", KTX_ORIENT_X_RIGHT)
-    ;
-    enum_<ktxOrientationY>("OrientationY")
-        .value("UP", KTX_ORIENT_Y_UP)
-        .value("DOWN", KTX_ORIENT_Y_DOWN)
-    ;
-    enum_<ktxOrientationZ>("OrientationZ")
-        .value("IN", KTX_ORIENT_Z_IN)
-        .value("OUT", KTX_ORIENT_Z_OUT)
-    ;
+        .value("TRANSCODE_ALPHA_DATA_TO_OPAQUE_FORMATS", KTX_TF_TRANSCODE_ALPHA_DATA_TO_OPAQUE_FORMATS);
+    enum_<ktxOrientationX>("OrientationX").value("LEFT", KTX_ORIENT_X_LEFT).value("RIGHT", KTX_ORIENT_X_RIGHT);
```

Comment:
Why has it run these `.value`s together? It hasn't in other cases. Can you prevent it here? This is much harder to read and understand than the original formatting with each .value on a separate line.

Reply:
We'll check whether there's an option for this, but please consider that even clang-format's configuration options are limited, so I cannot promise anything.